packages/frontend/src/components/SchedulersView/LogsView.tsx

```diff
@@ -0,0 +1,339 @@
+                                                    <Text color="white" span>
+                                                        {user?.firstName}{' '}
+                                                        {user?.lastName}
+                                                    </Text>
+                                                </Text>
+                                            </Stack>
+                                        }
+                                    >
+                                        <Text
+                                            fw={600}
+                                            lineClamp={1}
+                                            sx={{
+                                                overflowWrap: 'anywhere',
+                                                '&:hover': {
+                                                    textDecoration: 'underline',
+                                                },
+                                            }}
+                                        >
+                                            {item.name}
+                                        </Text>
+                                    </Tooltip>
+                                    <Text fz={12} color="gray.6">
+                                        {chartOrDashboard?.name}
+                                    </Text>
+                                </Stack>
+                            </Group>
+                        </Anchor>
+                    );
+                },
+                meta: {
+                    style: {
+                        width: 250,
+                    },
+                },
+            },
+            {
+                id: 'jobs',
+                label: 'Job',
+                cell: (item) => {
+                    return currentLogs(item, logs).length > 0 ? (
+                        <Box>
+                            <Group spacing="two">
+                                <Text fz={12} fw={500}>
+                                    All jobs
+                                </Text>
+                                <ActionIcon onClick={toggle} size="sm">
+                                    <MantineIcon
+                                        icon={IconChevronDown}
+                                        color="black"
+                                        size={13}
+                                    />
+                                </ActionIcon>
+                            </Group>
+                            <Collapse in={opened}>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
+                                    {handleScheduledDeliveryLogs(item, logs)[0]
+                                        .task.replace(/([A-Z])/g, ' $1')
+                                        .toLowerCase()}
+                                </Text>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
```

Comment:
same wrappers, can be one Stack or Text component with this set of props and two nested Text components with no props.